src/packages/cli/helpers/build.js

```diff
@@ -17,10 +35,14 @@ async function main() {
   await Promise.all([
     run('node ./helpers/copy-prisma-client.js'),
     run('tsc --build tsconfig.build.json', true),
+  ])
+
+  await Promise.all([
```

Comment:
Had to separate these steps, since the prismaClientPlugin needs prisma-client to be written to disk before it can work